userMap.json

```diff
@@ -19,6 +19,7 @@
     "sterprim": ["xctan"],
     "xeonacid": ["Xeonacid"],
     "xunop": ["Xunop"],
-    "xyenchi": ["XYenChi"]
+    "xyenchi": ["XYenChi"],
+    "qyl27": ["qyl27"]
```

Comment:
Please try to keep the file in alphabetical order.

Reply:
I'm sorry.